_posts/2019/2019-04-30-how-to-easily-manage-ssh-key-api-implementation.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: how to easily manage ssh keys - API implementation
+date: 2019-03-31
+header_image: public/ssh-key-management.jpg
+header_position: bottom
+header_overlay: true
+category: tech-stories
+tags: ["idoit", "ssh", "development", "tools"]
+authors: ["Carsten"]
+about_authors: ["cseeger"]
+---
+
+In the previous post [How to easily manage SSH keys](/blog/tech-stories/how-to-easily-manage-ssh-keys/) we discussed how to manage SSH keys.
+Now it's time to implement the backend.
```

Comment:
Would be good if you can explain this a bit to have more context: implement the backend to achieve what?